index.html

```diff
@@ -4,49 +4,218 @@
 <head>
   <meta charset='utf-8'>
   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-  <title>Page Title</title>
+  <title>Mobile clone page</title>
   <meta name='viewport' content='width=device-width, initial-scale=1'>
   <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link
-    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;500;700&display=swap"
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&familty=Inter:wght@500;700&family=Roboto:wght@400;500;700&display=swap&"
     rel="stylesheet">
   <script src='main.js'></script>
 </head>
 
 <body>
-  <header class="main-container">
-    <nav>
-      <div class="logo">
-        <a href="#">Logo</a>
+  <div class="cards-container">
+
+    <header class="card-start-screen">
+      <nav>
+        <div class="logo">
+          <a href="#">Logo</a>
+        </div>
+        <div class="menu">
+          <button class="menu-btn">
+            <img src="assets/Icon-Menu.png" alt="menu button">
+          </button>
+        </div>
+      </nav>
+      <section class="headline">
+        <div>
+          <h1>Hey There.
+            I’m Gabriel</h1>
+          <h2>I’am a Software Developer</h2>
+          <p class="dark">I can help you build a product , feature or website Look through some of my work and
+            experience! If you
+            like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        </div>
+        <div class="contact-info">
+          <ul>
+            <li><a href="#"> <img src="assets/Icon-GitHub.png" alt="github logo"></a></li>
+            <li><a href="#"> <img src="assets/Icon-Linkedin.png" alt="linkedin logo"></a></li>
+            <li><a href="#"> <img src="assets/Icon-Angelist.png" alt="angelist logo"></a></li>
+            <li><a href="#"> <img src="assets/Icon-Medium.png" alt="medium logo"></a></li>
+            <li><a href="#"> <img src="assets/Icon-Twitter.png" alt="twitter logo"></a></li>
+          </ul>
+        </div>
+      </section>
+    </header>
+
+    <section class="works">
```

Comment:
# Can you explain this better please? I am confused because:
- my cards-container class already displays cards in a grid layout (my main container its called cards-container)
- all my individual cards use flex display for their inner elements 

Thanks for the feedback!

Reply:
The main container for the `div` with class name "card" is `section` with class name "works" and the project requires you to use display: grid to position all the `div`s with class name "card". let me know if it's not clear thanks 👍

Reply:
# Thanks! 
I think I understand now, only the recent projects / work cards should be inside the grid, everything else outside.
is that correct?

Reply:
yes, 👍 all the cards should be inside a grid.

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: